pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -3149,102 +3149,6 @@ spec:
   taskRef:
```

Comment:
I am under the impression that this whole test and CloudEvent could be removed along with the removal of CloudEvent resources.

Reply:
yes, I think so